src/components/common/GProfiler/utils.ts

```diff
@@ -0,0 +1,141 @@
+import {
+    GPROFILER_API_ENDPOINT__GENE_LIST_FUNCTIONAL_ENRICHMENT,
+    GPROFILER_LINK_MAX_LENGTH,
+} from './constants';
+
+import { FeatureMetadata, FeatureMetricTable } from './types';
+
+export const getMetricTable = (featureMetadata: FeatureMetadata) => {
+    return featureMetadata.genes.map((gene: string, idx: number) => {
```

Comment:
This is a bit hairy...
Wondering how it could be simplified... could you operate on `R.zip(featureMetadata.genes, featureMetadata.metrics)` ? Maybe name the reducing function?

Reply:
Ok I named the reducing function